{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "384a64d1_e360b707",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 9
      },
      "lineNbr": 0,
      "author": {
        "id": 1000350
      },
      "writtenOn": "2022-09-21T15:31:33Z",
      "side": 1,
      "message": "Checking CI failure,\nseems unrelated to this commit\n./redfish-core/include/../lib/managers.hpp:1282:5: error: an exception may be thrown in function \u0027~GetPIDValues\u0027 which should not throw exceptions [bugprone-exception-escape,-warnings-as-errors]\n\n",
      "revId": "f805bce64ce94e039393024de8798c39ecd1a890",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}